extension-manifest-v3/src/content_scripts/adblocker-scriptlets.js

```diff
@@ -0,0 +1,5 @@
+// Should only be needed on Safari:
+// the tabId of the initial chrome.webNavigation.onCommitted
+// is not reliable. When opening bookmarks, it can happen that
+// the event is associated with a tabId of 0.
+chrome.runtime.sendMessage({ action: 'onCommitted' });
```

Comment:
May it lead to double-injections?

Reply:
no, as this is loaded for Safari only and on Safari we listen to this message instead of listening to webNavigation https://github.com/ghostery/ghostery-extension/blob/f4ca329c15edaf073bd7297e6e2dfaddbb710d0e/extension-manifest-v3/src/background/adblocker.js#L303